src/DynamoCore/Search/SearchElements/CustomNodeSearchElement.cs

```diff
@@ -30,6 +30,17 @@ private set
             }
         }
 
+        public string CustomizationAssembly
+        {
+            get
+            {
+                string customizationPath = System.IO.Path.GetDirectoryName(Path);
+                customizationPath = System.IO.Directory.GetParent(customizationPath).FullName;
+                customizationPath = customizationPath + Dynamo.UI.Configurations.PackageCustomizationAssemblyPostfix;
+                return customizationPath;
+            }
+        }            
```

Comment:
Oops, my previous comment was removed, but here it is...

Please use `Directory.GetParent(customizationPath).FullName` instead of `ToString`. And instead of appending directory names like that, we should really do this:

```csharp
customizationPath = Path.Combine(customizationPath, "bin", "Package.dll");
```

But come to think about it... we might not want to add this property at all (having a name like `Package.dll` that does not exist sounds weird, and the purpose may not immediately be obvious at this point). This code can be added to the caller of `CustomizationAssembly` property below in `NodeSearchElementViewModel.cs`:

```csharp
protected override ImageSource GetIcon(string fullNameOfIcon)
{
    string customizationPath = System.IO.Path.GetDirectoryName(Model.Path);
    customizationPath = System.IO.Directory.GetParent(customizationPath).FullName;
    customizationPath = Path.Combine(customizationPath, "bin", "Package.dll");

    var iconRequest = new IconRequestEventArgs(customizationAssemblyPath, ...);
}
```